src/app/api/trivia/route.ts

```
@@ -140,422 +178,580 @@ async function getTriviaQuestions(
       // Route based on action parameter with enhanced error handling
       switch (action) {
-        case 'questions':
+        case "questions":
           // Get trivia questions (default behavior)
-          const result = await getTriviaQuestions(userId, authenticatedSupabase, triviaContext);
-          tracker.complete(true, { action, questionCount: Array.isArray(result) ? result.length : 0 });
+          const result = await getTriviaQuestions(
+            userId,
+            authenticatedSupabase,
+            triviaContext,
+          );
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)